CreeDictionary/tests/DatabaseManager_tests/test_xml_importer_regressions.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from API.models import Wordform
+from tests.conftest import migrate_and_import
+
+
+@pytest.mark.django_db
+def test_import_xml_common_analysis_definition_merge(shared_datadir):
+    migrate_and_import(shared_datadir / "crkeng-common-analysis-definition-merge")
+
+    query_set = Wordform.objects.filter(text="nipa")
+
+    kill_him_inflections = []
+    for inflection in query_set:
+        for definition in inflection.definitions.all():
+            if "Kill" in definition.text:
+                kill_him_inflections.append(inflection)
+
+    assert len(kill_him_inflections) == 1
+    kill_him_inflection = kill_him_inflections[0]
+    assert kill_him_inflection.pos == "V"
+
+
+@pytest.mark.django_db
+def test_import_pipon_of_different_word_classes(shared_datadir):
+    # https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/190
+    # Issue description: search results for some inflected form of word pipon is not showing up
+    # Cause: pipon lemmas wrongly marked as "as-is" in the database when the xml actually provided enough resolution
+    # on the word classes (VII and NI)
+
+    # The Cree word pipon has two entries in the test xml, one's word class is VII and the other's is NI
+    migrate_and_import(shared_datadir / "crkeng-pipon-of-different-word-classes")
+
+    # todo: let `migrate_and_import` report success/ambiguity/no-analysis count so that further tests to the importer
+    #   can be easier constructed. e.g. in this case we'll only need to assert `success == 2`
```

Comment:
I proposed this here and I want to do it later in a different PR. Our importer already has beautify output to stdout:

```
xx entries have (lemma, pos, lc) duplicate to others. Their definition will be merged
xx (xml_lemma, pos, lc) tuples extracted
xx (lemma, pos, lc) found proper lemma analysis
There are xx (lemma, pos, lc) that the fst can not give any analyses.
There are xx (lemma, pos, lc) that do not have proper lemma analysis by fst
There are xx (lemma, pos, lc) that have ambiguous lemma analyses
```
When I was debugging the issue, stdout told me 
"**0** Found proper analysis. There are **2** that have ambiguous lemma analyses"
 which made me realize the issue immediately. 
And I knew I fixed the issue as it turned:
"**2** Found proper analysis. There are **0** that have ambiguous lemma analyses"

So returning the digits will help in building tests.